app/src/main/res/layout/activity_cadastrarendereco.xml

```diff
@@ -4,6 +4,162 @@
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <EditText
+            android:id="@+id/edittextenderecocliente"
+            android:layout_width="350dp"
+            android:layout_height="50dp"
+            android:layout_marginTop="50dp"
+            android:background="@drawable/edittextcriarconta"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/txttituloendereco"
+            android:textColor="#000"
+            android:hint="@string/hintedittextcadastroendereco"
+            android:textColorHint="#717171"
+            android:inputType="text" />
+
+        <EditText
+            android:id="@+id/edittextenderecoclientecomp"
+            android:layout_width="350dp"
+            android:layout_height="50dp"
+            android:layout_marginTop="40dp"
+            android:background="@drawable/edittextcriarconta"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/edittextenderecocliente"
+            android:textColor="#000"
+            android:hint="@string/hintedtextcadastroenderecocomp"
+            android:textColorHint="#717171"
+            android:inputType="text"/>
+
+        <!--confirm button-->
+        <androidx.cardview.widget.CardView
+            android:id="@+id/edittextenderecoconfirmar"
+            android:layout_width="350dp"
+            android:layout_height="50dp"
+            android:layout_marginTop="50dp"
+            app:cardBackgroundColor="#FFC75A"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/edittextenderecoclientecomp">
+
+            <!--Animation and text base-->
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <!-- text animation-->
+                <com.airbnb.lottie.LottieAnimationView
+                    android:id="@+id/animationenderecoconfirm"
+                    android:layout_width="200dp"
+                    android:layout_height="match_parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:lottie_loop="true"
+                    app:lottie_autoPlay="true"
+                    app:lottie_rawRes="@raw/coffeeloading"/>
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Confirmar"
+                    android:textColor="#fff"
+                    android:textSize="30dp"
```

Comment:
Utilizar @string/ para referenciar um texto para que manutença futura seja mais pratico e rápido
Utilizar sp ao invés de dp quando for informar o tamanho do texto